frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/volume-wrapper.ts

```diff
@@ -116,6 +117,8 @@ export class VolumeWrapper extends ObjectWithTypePropertyWrapper<
 
   getName = () => this.get('name');
 
+  isEmpty = () => _.isEmpty(this.data);
```

Comment:
this can be reused by all wrappers. Please move it to wrapper component.

Reply:
good idea! +1